src/utils/TagUtils.ts

```diff
@@ -1,5 +1,6 @@
 import Log from '../libraries/Log';
 import { TagsObject } from '../models/Tags';
+import _ from "lodash";
```

Comment:
Need remove and use something very small or write our own code for what we use here.